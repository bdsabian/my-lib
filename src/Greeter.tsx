import * as React from "react";

interface GreeterProps {
  name: string;
}
const Greeter = ({ name }: GreeterProps) => {
  return <span>Hello {name}</span>;
};

export default Greeter;
