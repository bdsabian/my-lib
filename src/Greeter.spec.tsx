import * as React from "react";
import * as renderer from "react-test-renderer";

import Greeter from "./Greeter";

test("Greeter shows the name", () => {
  const component = renderer.create(<Greeter name="Brendon" />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
