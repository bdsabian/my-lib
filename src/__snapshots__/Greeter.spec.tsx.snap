// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeter shows the name 1`] = `
<span>
  Hello 
  Brendon
</span>
`;
